.github/workflows/pulumi-cli-dev-version.yml

```
@@ -0,0 +1,57 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: Update latest-dev-version
+on:
+  repository_dispatch:
+    types:
+      - pulumi-cli-dev-version
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: update-latest-dev-version
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "pulumi/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen dev version pulumi@${{ github.event.client_payload.dev_version}}"
+          pr_body: "Automated PR"
+          pr_label: "automation/pulumi-cli-docs,automation/merge"
+          github_token: ${{ secrets.PULUMI_BOT_TOKEN }}
+  build-pulumi-cli-docs:
```

Should this be this?

```suggestion
  update-latest-dev-version:
```